### ISSUE TITLE
Using the compositer in blender breaks the spout output. 

### ISSUE THREAD

**Issue by jacobmesick** [maybites/blender.script.spout#44]
Hey guys, 

First off let me thank you for such an amazing plug-in for blender. This allows me to do some fantastic projects in Resolume in real time. Without this plugin I would rely on looping animations and less inspiration “at the moment” type art I am making, again thank you. 

I wanted to just mention that I noticed that the  plug-in does not like the compositer to be used when spouting the window output to other programs. I will make sure to list out my troubleshooting of this very soon to give you the full picture. The system stops broadcasting when the compositer is activated.  

Thank you for your time.

**Comment by maybites** [comment]
Test if this script works with compositor:

https://docs.blender.org/api/master/gpu.html#rendering-the-3d-view-into-a-texture

if it does, than my implementation has a problem, if it doesn't than it is a bug in blender..

**Comment by jacobmesick** [comment]
I am sorry I wish I had the knowledge and understanding to test this script. As promised however I wanted to give you a little more to go on. I know this is just something as a "gee wiz." you guys have already made something amazing and it is much appreciated. 

Too keep it simple I used the default cube scene and activated the "use nodes check box in the compositor window, then I added the brightness and contrast node between the render layers node and the Composite node. Once this is complete I turned on "viewport shading" in the layout tab and selected compositor always in the drop down box. as soon as the broadcast is made with the texture sharing option turned on it shuts down blender. not a real big issue I don't really need the compositor for what I am doing, again just a nice to have but not a need to have.

**Comment by jacobmesick** [comment]
Okay I think I was able to figure out that script and try it out. The system puked at the same point when I activated the compositor which means that it is blender that Causes this issue. Thank you.
